package.json

```diff
@@ -28,13 +28,16 @@
     "test": "jest",
     "test.prod": "yarn lint && yarn test --coverage --maxWorkers=2"
   },
+  "resolutions": {
```

Comment:
Are we going to need to add this to downstream dependencies? We've had a bunch of issues with forks causing broken stuff because we did not add the resolution to each package.json

Reply:
I think we should just bump the version every where the http spec is being used. Then we don’t run into the resolution issue.

Reply:
We shouldn't need this resolution here in general? I think it can be removed. Resolutions is kind of case by case basis depending on the package involved - assuming there are no breaking changes, generally they should be set in end products imho. In this case I think we'd need to add a resolution to studio (for studio desktop), app-gen3, and stoplight/cli.

Reply:
Or what tom said, and no resolutions needed.

Reply:
Ok, I remember something about Prism binaries breaking and somebody else struggling to NPM behind a corp firewall for some reason, but if you think relying on resolved is ok then sure thing. Maybe @XVincentX remembers the specifics.

Reply:
In general packages coming from GitHub can be a pain because its scripts aren't executed — so if some of these have something to compile/move/anything, it won't work.

I do not think there should be a problem in this case, but in general less we do this the better.